index.bs

```diff
@@ -1126,10 +1129,10 @@ An {{XRPose}} describes a position and orientation in space relative to an {{XRS
 };
 </pre>
 
-The <dfn attribute for="XRPose">transform</dfn> attribute describes the {{XRRigidTransform}} between two {{XRSpace}}.
+The <dfn attribute for="XRPose">transform</dfn> attribute describes the position and orientation relative to the underlying {{XRSpace}}.
```

Comment:
There's some potential confusion here since the getPose definition uses sourceSpace and destinationSpace in its arguments, and it's not obvious which of these is the "underlying" space. 

How about using `relative to the reference {{XRSpace}}` here, and changing the getPose definition to match? Something like this?

```
XRPose? getPose(XRSpace space, XRSpace referenceSpace);
...
When the <dfn method for="XRFrame">getPose(|space|, |referenceSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
...
  1. [=Populate the pose=] of |space| in |referenceSpace| at the time represented by |frame| into |pose|.
 ```

Not sure what the best name is for the first argument. The proposal here is roughly following the explainer which uses `getPose(xrSpace, xrReferenceSpace)`. Other options would be `posedSpace`, `objectSpace`, `trackedSpace`?

Reply:
Oh, I like this. This better clarifies the source/destination thing too.